### PR TITLE
Button functionality to display booking page

### PR DESCRIPTION
**Pull request by tanishalatoya**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] New feature (non-breaking, but doesn't add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Detailed Description
I added a button to the user's dashboard that allows them to go to a page with all the destinations options to eventually book a new trip there.

### Why is this change required? What problem does it solve?
This allows a user to not only see trips they've had/will have, but also allows them to book a new trip.

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?
None at this time.

### Files modified:
- [ ] index.html
- [ ] base.scss
- [X] styles.scss
- [X] index.js
- [X] domUpdates.js
- [ ] Other files:

Instructors: @khalidwilliams  @hannahhch

### CODE REVIEW

File: src/css/styles.scss [hunk 1]
```diff
@@ -169,3 +169,22 @@ form input {
   justify-content: space-around;
   overflow: scroll;
 }
+
+//DESTINATION BOOKING PAGE
```
Comment (ttarlov) [inline src/css/styles.scss]: Awesome to have notes like that clarifying styles!

File: src/index.js [hunk 2]
```diff
@@ -29,13 +29,12 @@ const tripsData = fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/191
...
-signInButton.on('click', signInTraveler);
-adminLogIn.on('click', domUpdates.displayAdminLogInScreen);
-adminLogIn.on('click', domUpdates.createAdminSignInButton);
+$('#sign-in-submit').on('click', signInTraveler);
+$('#admin-log-in').on('click', domUpdates.displayAdminLogInScreen);
+$('#admin-log-in').on('click', domUpdates.createAdminSignInButton);
+
 
 $(document).ready(() => {
   Promise.all([travlersData, destinationsData, tripsData])
```
Comment (ttarlov) [inline src/index.js]: Promise.all is great to use for multiple dependant fetches. Very cool.

**Comment by ttarlov** [comment]
I wish my code was this clean. Very nice job Tanisha!

File: src/css/styles.scss (hunk 1 above)
Comment (tanishalatoya) [reply src/css/styles.scss]: Yeah, this also helps guide me when I'm thinking about where I need to put the rules for a certain style.

**Comment by tanishalatoya** [comment]
> I wish my code was this clean. Very nice job Tanisha!

Thanks Taras, but I definitely have so more cleaning up to do. Thanks for taking the time out to review my code!

**Pull request merged**